packages/documentation-site/patternfly-docs/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -27,22 +27,29 @@ Think of each row in a data list as a container for some formatted content. In P
 * **[Split](/layouts/split):** Distributes content evenly with a main content area in the center.
 * **[Flex](/layouts/flex):** Enables more customization control over the alignment and spacing options provided in the other layouts.
 
-PatternFly offers 2 components for displaying large data sets: data lists and [tables](/components/table/design-guidelines). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
+Data list toolbars should only contain actions that directly impact the data list. Any actions that are related to the overall page and its context (or impact the general page but not the data list directly) should be placed outside of the data list, at the top right of the page at the header level. This may include actions that edit the page header, actions that launch related windows, and so on.
 
 Data lists can also appear in primary-detail views. Visit the [primary-detail guidelines](/patterns/primary-detail/design-guidelines) to learn more about the functionality. 
 
 ### When to use a data list vs. a table
+PatternFly offers 2 components for displaying large data sets: data lists and [tables](/components/table/design-guidelines). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
+
 **Use a data list when**:
 * The information you want to display is not easily structured into a tabular format.
 * Content displayed varies between rows and you want a more flexible layout. 
 * You plan to embed rich content like a chart or an image into a row.
 
-**Use a table when**:
+**Use a [table](/components/table) when**:
 * The information you want to display is easily structured into a tabular format.
 * The clarity of your content would benefit from well defined columns and headings.
 
+### When to use a data list vs. a card
+Instead of a data list, you may choose to use [card views](/components/card/design-guidelines#card-views). Card views and data lists have similar properties, but information in a card view is chunked into a grid of individual cards. 
+
+In choosing between a data list and a card view, consider the type of data that will be displayed and the format that best suits that data. If you cannot easily fit all of the data that needs to be displayed into a card, a data list might be a better solution.
+
 ## Variations
-PatterFly supports 2 main types of data lists:
+PatternFly supports 2 main types of data lists:
 
 1. **[Compact data list](#compact-spacing):** Use when you want to show as much data per page as possible.
 2. **[Default data list](#default-spacing):** Use when you don’t have to minimize paging.
```

Comment:
you didn't edit this but I just notice there's a space missing underneath the variation bullets where it says "See when to use compact vs. default spacing for more information"

There's a space missing between "spacing" and "for" at the end of the sentence

![image](https://github.com/patternfly/patternfly-org/assets/51165119/1f5f1e33-0344-4157-b742-920840b59b99)

Reply:
oo great catch thank you!